compose/project.py

```diff
@@ -251,7 +275,7 @@ def initialize_volumes(self):
     def down(self, remove_image_type, include_volumes):
         self.stop()
         self.remove_stopped(v=include_volumes)
-        self.remove_network()
+        self.remove_default_network()
```

Comment:
I think this needs to remove all networks, not just the default.  Since there's a TODO on not removing external, I guess this could be a TODO as well